go.mod

```diff
@@ -1,20 +1,21 @@
 module sigs.k8s.io/cluster-api-provider-openstack
 
-go 1.13
+go 1.15
```

Comment:
golang should be 1.16.

Document was updated ;)
https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#minimum-go-version

Reply:
Done